app/main/views/briefs.py

```diff
@@ -183,6 +220,43 @@ def update_brief_remove_user(brief_id, user):
     return jsonify(briefs=brief.serialize(with_users=True)), 200
 
 
+@main.route('/briefs/<int:brief_id>/suppliers/<int:supplier_code>', methods=['DELETE'])
+def update_brief_remove_rfx_seller(brief_id, supplier_code):
+    updater_json = validate_and_return_updater_request()
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    # Assign the current seller data to separate variable.
+    # Check the keys of the new variable to find if the supplier is in the dictionary.
+    # If found, delete. Otherwise raise the validation error.
+    sellers = brief.data['sellers']
+
+    for key in sellers.keys():
+        try:
+            if key == str(supplier_code):
+                del sellers[key]
+        except KeyError:
+            raise ValidationError("No supplier found: " + supplier_code)
+    # Amend the brief's data to reflect the shortened list of suppliers.
+    brief.data['sellers'] = sellers
+    audit = AuditEvent(
+        audit_type=AuditTypes.update_brief,
```

Comment:
same comment as above

Reply:
This has been updated in my latest commit.